docs/sources/experiments/prediction-ranking.md

```diff
@@ -0,0 +1,50 @@
+# Prediction Ranking
+
+The predictions tables in the `train_results` and `test_results` schemas contain several different flavors of rankings, covering absolute vs percentile ranking and whether or not ties exist.
+
+## Ranking columns
+
+| Column name | Behavior |
+| ----------- | ------- |
+| rank_abs_with_ties | Absolute ranking, with ties. Ranks will skip after a set of ties, so if two entities are tied at rank 3, the next entity after them will have rank 5. |
+| rank_pct_with_ties | Percentile ranking, with ties. Percentiles will skip after a set of ties, so if two entities out of ten are tied at 0.1 (tenth percentile), the next entity after them will have 0.3 (thirtieth percentile) |
```

Comment:
might be good to mention if the percentiles are calculated to a certain precision?